DESCRIPTION

```diff
@@ -8,7 +8,10 @@ Authors@R:
     person("Marek", "Rogala", role = "aut", email = "marek@appsilon.com"),
     person("Appsilon Sp. z o.o.", role = "cph", email = "opensource@appsilon.com")
   )
-Description: It provides all controls from Microsoft Fluent UI.
+Description:
+  A rich set of UI components for building Shiny applications,
```

Comment:
Wouldn't we prefer to keep it here that those components come from Microsoft Fluent UI?